<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>874</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MEASUREMENT QT</string>
  </property>
  <widget class="QGroupBox" name="Script_groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>871</width>
     <height>751</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="title">
    <string>Script</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btn_script_load">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/load.png</normaloff>../img/load.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_script_update">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../img/update.png</normaloff>../img/update.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_script_path">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">qproperty-cursorPosition: 0;
border: 1px solid transparent;</string>
    </property>
    <property name="text">
     <string>Script Path</string>
    </property>
    <property name="frame">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="tbl_script">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>851</width>
      <height>671</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial Black</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QLabel" name="label_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>LG Smart UI SemiBold</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MESUREMENT</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Run_Script">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>60</y>
     <width>181</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #42f566;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>RUN SCRIPT</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_69">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid transparent;
color: black;</string>
   </property>
   <property name="text">
    <string>Measure Target : </string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbox_project">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>60</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
selection-background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
border: 1px solid transparent;
color: white;
}

QComboBox QAbstractItemView
{
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #565656, stop: 0.1 #525252, stop: 0.5 #4e4e4e, stop: 0.9 #4a4a4a, stop: 1 #464646);
selection-background-color: rgb(170, 170, 255);
color: white;
}

QComboBox:hover
{
border: 4px solid green;
color: white;
}

QComboBox::drop-down
{
border-width: 0px;
}
</string>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>  MeasureOPU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>  MeasureDynamoTVTC</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_Result_Folder">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>60</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Black</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
	color: black;
	border-radius: 5px;
	border: 1px solid;
}
QPushButton::pressed{
background-color : blue; color: blue
}
QPushButton::hover{
background-color: lightblue;
}</string>
   </property>
   <property name="text">
    <string>Open Result</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
